arch/arm/soc/atmel_sam/sam4s/soc_pinmap.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2017 Justin Watson
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief Atmel SAM4S MCU pin definitions.
+ *
+ * This file contains pin configuration data required by different MCU
+ * modules to correctly configure GPIO controller.
+ */
+
+#ifndef _ATMEL_SAM_SOC_PINMAP_H_
+#define _ATMEL_SAM_SOC_PINMAP_H_
+
+#include <soc.h>
+
+/* Universal Asynchronous Receiver Transmitter (UART) */
+
+#define PIN_UART0_RXD {PIO_PA9A_UART0_URXD0,  PIOA, ID_PIOA, SOC_GPIO_FUNC_A}
+#define PIN_UART0_TXD {PIO_PA10A_UART0_UTXD0, PIOA, ID_PIOA, SOC_GPIO_FUNC_A}
+
+#define PINS_UART0 {PIN_UART0_RXD, PIN_UART0_TXD}
+
+#define PIN_UART1_RXD {PIO_PB2A_UART1_URXD1,  PIOA, ID_PIOA, SOC_GPIO_FUNC_A}
```

Comment:
This pin is on PIOB, not PIOA. The correct definition is:
```
#define PIN_UART1_RXD {PIO_PB2A_URXD1,  PIOB, ID_PIOB, SOC_GPIO_FUNC_A}
```
The same applies to PIN_UART1_TXD.